api/search.go

```diff
@@ -155,24 +155,58 @@ func CreateRequests(w http.ResponseWriter, req *http.Request, validator QueryPar
 	contentTypes := defaultContentTypes
 	if contentTypesParam != "" {
 		contentTypes = strings.Split(contentTypesParam, ",")
-		disallowed, err := validateContentTypes(contentTypes)
-		if err != nil {
-			log.Warn(ctx, err.Error(), log.Data{"param": ParamContentType, "value": contentTypesParam, "disallowed": disallowed})
+		disallowed, er := validateContentTypes(contentTypes)
+		if er != nil {
+			log.Warn(ctx, er.Error(), log.Data{"param": ParamContentType, "value": contentTypesParam, "disallowed": disallowed})
 			http.Error(w, fmt.Sprint("Invalid content_type(s): ", strings.Join(disallowed, ",")), http.StatusBadRequest)
```

Comment:
why `er` not `err`?

Reply:
Because the golinter finds err to be a shadow variable. 
Do you want me to ignore the issue `//nolint:golint,unused` ?

Reply:
could just call it something more distinct like `validationErr`